Source/Core/DolphinQt/RenderWidget.h

```diff
@@ -11,6 +11,7 @@
 
 class QMouseEvent;
 class QTimer;
+class DragEnterEvent;
```

Comment:
This is missing a Q prefix

Reply:
Kinda surprised this compiles if its the wrong prefix. Is it even needed in here?

Reply:
I lifted this code from MainWindow.h. It was included there with no Q prefix.

Reply:
The inclusion of QWidget is all that's required for QDragEnterEvent and QDropEvent to resolve.  I only included it as it was included in MainWindow.h which is where I lifted the code from. It compiled as it's not even required.